pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -290,6 +290,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	}
 
 	cm.topologyManager, err = topologymanager.NewManager(
+		context.TODO(),
```

Comment:
Would it be better to pass ctx from the caller? It looks like it's available there.